### PR TITLE
DEV-1756_style_preview_are_blank

### PR DESCRIPTION
**Pull request by gurjittoptal**
Please fill out each section even if it's just with "N/A"

#### Plan? (if this draft/incomplete indicate what you intend to do and how)
Styles in frontend app broke after response for /api/styles changed

#### What's this PR do?
makes styles work again in dashboard app

#### Why are we doing this? How does it help us?
critical bug

#### How should this be manually tested?
On dashboard app, styles card should not be blank and should be clickable

#### Have automated unit tests been added?
No.

#### How should this change be communicated to end users?
na

#### Are there any smells or added technical debt to note?
na

#### Has this been documented? If so, where?
na

#### What are the relevant tickets?
**(Note: Please use syntax [JIRA-123] to link any relevant tickets)**
[DEV-1756]

#### Are there next steps? If so, what? Have tickets been opened for them?

[DEV-1756]: https://news-revenue-hub.atlassian.net/browse/DEV-1756?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: spa/src/components/content/Content.js [hunk 1]
```diff
@@ -33,7 +33,18 @@ function Content() {
...
+          for (let d = 0; d < data.length; d++) {
+            let styleReformat = data[d].styles;
+            styleReformat['id'] = data[d].id;
+            styleReformat['revenue_program'] = data[d].revenue_program;
+            styleReformat['name'] = data[d].name;
+            styleReformat['used_live'] = data[d].used_live;
+            allStyles.push(styleReformat);
+          }
+          setStyles(allStyles);
+        },
```
Comment (benjaminEwhite) [inline spa/src/components/content/Content.js:47]:
```suggestion
        onSuccess: ({ data }) => {
          setStyles(data.map(({styles, id, revenue_program, name, used_live}) => {
            return {...styles, id, revenue_program, name, used_live}
          }));
        },
```

**Review by benjaminEwhite — CHANGES_REQUESTED** [review]
Requested 2 small stylistic changes, but implementation looks good.

File: spa/src/components/content/Content.js (hunk 1 above)
Comment (gsclb) [reply spa/src/components/content/Content.js]: updated

File: spa/cypress/fixtures/styles/list-styles-1.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
     "id": 1,
     "name": "My Test Styles 01",
     "used_live": true,
+    "styles":{
```
[the remark this answers is not in the document]
Comment (gsclb) [reply spa/cypress/fixtures/styles/list-styles-1.json:6]: agreed. changes done.

**Review by benjaminEwhite — APPROVED** [review]

**Pull request merged**